src/configs/config.yaml

```diff
@@ -0,0 +1,5 @@
+key_path:  "/etc/config/key.pem"
+cert_path: "/etc/config/cert.pem"
+db_path: "/etc/config/certs.db"
```

Comment:
Following my comment on `charmcraft.yaml`, we should use the directories defined there. Those can be variables in charm.py and rendered using jinja templates

Reply:
I think using storage mounts makes sense to do in a separate PR, it has some edge cases like handling mounting and unmounting that should be reviewed in detail.